.github/workflows/examples-nxp.yaml

```diff
@@ -150,7 +150,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: ghcr.io/project-chip/chip-build-rw61x:74
```

Comment:
why the version rollback? could we use a -nxp 74 build?

I generally would not expect a version rollback in any circumstance. If we need to fix some build image, we should fix it and roll the version forward.

Reply:
Hi Andy,
Thanks for the comment, I fix the docker image version